Source/UnitTests/Common/StringUtilTest.cpp

```diff
@@ -3,6 +3,7 @@
 // Refer to the license.txt file included.
 
 #include <gtest/gtest.h>
+#include <gmock/gmock.h>
```

Comment:
We don't have gmock in Externals...but, why don't you try `EXPECT_STREQ` to compare the string results?